testapp/src/main/res/layout/settings_menu_base_activity.xml

```diff
@@ -0,0 +1,74 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:padding="@dimen/medium_16"
+  android:orientation="vertical">
+
+  <androidx.appcompat.widget.AppCompatTextView
+    android:id="@+id/textInfo"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content" />
+
+  <RelativeLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/small_8">
+
+    <androidx.appcompat.widget.AppCompatButton
+      android:id="@+id/buttonCancel"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_alignParentStart="true"
+      android:text="@string/action_cancel" />
```

Comment:
On iOS we don't have a back button, so maybe on Android this button is not necessary. Just confirm with the team ;)

Reply:
Don't worry. Cancel button is necessary :)

Reply:
Thanks for your concerns, I have added screenshots based on my latest commits.